torchci/rockset/commons/__sql/weekly_force_merge_stats.sql

```diff
@@ -1,218 +1,170 @@
-WITH issue_comments AS (
-  SELECT
-    issue_comment.user.login,
-    issue_comment.author_association,
-    issue_comment.body,
-    issue_comment.issue_url,
-    issue_comment.html_url,
-    issue_comment.created,
-    CAST(
-      SUBSTR(
-        issue_comment.issue_url,
-        LENGTH(
-          'https://api.github.com/repos/pytorch/pytorch/issues/'
-        ) + 1
-      ) AS INT
-    ) AS pr_num
-  FROM
-    commons.issue_comment
-  WHERE
-    (
-      issue_comment.body LIKE '%pytorchbot merge%'
-      OR issue_comment.body LIKE '%pytorchmergebot merge%'
-    )
-    AND issue_comment.user.login NOT LIKE '%pytorch-bot%'
-    AND issue_comment.user.login NOT LIKE '%facebook-github-bot%'
-    AND issue_comment.user.login NOT LIKE '%pytorchmergebot%'
-    AND issue_comment.issue_url LIKE '%https://api.github.com/repos/pytorch/pytorch/issues/%'
-),
-all_merges AS (
-  SELECT
-    DISTINCT m.skip_mandatory_checks,
-    LENGTH(m.failed_checks) AS failed_checks_count,
-    LENGTH(m.ignore_current_checks) AS ignored_checks_count,
-    LENGTH(m.pending_checks) AS pending_checks_count,
-    m.ignore_current,
-    m.is_failed,
-    m.pr_num,
-    m.merge_commit_sha,
-    max(c.created) AS time,
-  FROM
-    commons.merges m
-    INNER JOIN issue_comments c ON m.pr_num = c.pr_num
-  WHERE
-    m.owner = 'pytorch'
-    AND m.project = 'pytorch'
-    AND m.merge_commit_sha != '' -- only consider successful merges
-    AND m._event_time >= PARSE_DATETIME_ISO8601(: startTime)
-    AND m._event_time < PARSE_DATETIME_ISO8601(: stopTime)
-  GROUP BY
-    m.skip_mandatory_checks,
-    m.failed_checks,
-    m.ignore_current,
-    m.is_failed,
-    m.pr_num,
-    m.merge_commit_sha,
-    m.ignore_current_checks,
-    m.pending_checks
-),
--- A legit force merge needs to satisfy one of the two conditions below:
--- 1. skip_mandatory_checks is true (-f) and failed_checks_count > 0 (with failures) or pending_checks_count > 0 (impatience).
---    Under this condition, if a force merge (-f) is done when there is no failure and all jobs have finished, it's arguably
---    just a regular merge in disguise.
--- 2. ignore_current is true (-i) and is_failed is false (indicating a successful merge) and ignored_checks_count > 0 (has failures).
---    As -i still waits for all remaining jobs to finish, this shouldn't be counted toward force merge due to impatience.
---
--- If none applies, the merge should be counted as a regular merge regardless of the use of -f or -i. We could track that
--- (regular merges masquerading as force merges) to understand how devs use (or abuse) these flags, but that's arguably a
--- different case altogether.
-merges_identifying_force_merges AS (
-  SELECT
-    IF(
-      (
-        skip_mandatory_checks = true
-        AND (
-          failed_checks_count > 0
-          OR pending_checks_count > 0
+-- gets percentage of total force merges, force merges with failures, and force merges without failures (impatient)
```

Comment:
Is this change expected? As I see lots of differences between them and the before version has more comments :)